ignite/cmd/scaffold_query.go

```diff
@@ -16,8 +16,9 @@ const (
 // NewScaffoldQuery command creates a new type command to scaffold queries.
 func NewScaffoldQuery() *cobra.Command {
 	c := &cobra.Command{
-		Use:     "query [name] [request_field1] [request_field2] ...",
-		Short:   "Query for fetching data from a blockchain",
+		Use:     "query [name] [request_field1] [request_field2:field2_type] ...",
+		Short:   "Query for fetching data from a blockchain\n",
+		Long:    "Query for fetching data from a blockchain\nFor detailed type information use ignite scaffold type --help",
```

Comment:
If you agree, this would keep consistency within multi-line long descriptions:
```suggestion
		Long:    `Query for fetching data from a blockchain.
		
For detailed type information use ignite scaffold type --help.`,
```